tool/stolonboot/boot.go

```diff
@@ -16,14 +16,13 @@ package main
 
 import (
 	"encoding/base64"
-	"io"
-	"os"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/gravitational/rigging"
 	"github.com/gravitational/trace"
 )
 
-func bootCluster(sentinels int, proxies int, password string) error {
+func bootCluster(sentinels, proxies, rpc int, password string) error {
```

Comment:
I think this is generally the worse option, makes all future refactoring harder. Would prefer to keep types per arg.